CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # What's New?
 
+## 1.2.3
+Bug fixes:
+- CTest status bar button text appears malformed. [#911] (https://github.com/microsoft/vscode-cmake-tools/issues/911)
```

Comment:
there should not be a space between `[]` and `()`.  If you put a space there, the hyperlink will not show up correctly.  Can you please fix this and the other ones below (including for 1.2.2.)?

Reply:
Done.